operation.go

```diff
@@ -118,7 +118,7 @@ func (operation *Operation) ParseComment(comment string, astFile *ast.File) erro
 var paramPattern = regexp.MustCompile(`(\S+)[\s]+([\w]+)[\s]+([\S.]+)[\s]+([\w]+)[\s]+"([^"]+)"`)
 
 // ParseParamComment parses params return []string of param properties
-// E.g. @Param	queryText		form	      string	  true		        "The email for login"
+// E.g. @Param	queryText		formData	      string	  true		        "The email for login"
```

Comment:
why this change can address #422 ?

Reply:
paramType 'form' is not supported, but in you sample give a paramType "form", this can result `swag init` failed with error "paramType form not supported"